src/components/tables/FieldSelection/Rows.tsx

```diff
@@ -104,7 +119,17 @@ function Row({ columns, row }: RowProps) {
             }}
         >
             <TableCell sx={getStickyTableCell()}>
-                <Typography>{row.field}</Typography>
+                <Typography
+                    style={
+                        actionRestricted ||
+                        (row.selectionType !== 'default' &&
+                            row.selectionType !== null)
+                            ? { fontWeight: 700 }
+                            : { fontStyle: 'italic' }
+                    }
```

Comment:
Since this might change pretty frequently I think `sx` should be good here.

Can you throw this into theme or something and then share this between the two tables in this and `Schema/Rows`